src/core/Microsoft.PowerToys.Settings.UI/Views/PowerRenamePage.xaml.cs

```diff
@@ -14,128 +14,16 @@ namespace Microsoft.PowerToys.Settings.UI.Views
 {
     public sealed partial class PowerRenamePage : Page
     {
-        public PowerRenameViewModel ViewModel { get; } = new PowerRenameViewModel();
+        public PowerRenameViewModel ViewModel { get; set; }
 
         private const string POWERTOYNAME = "PowerRename";
```

Comment:
nit: This can be removed too.

Reply:
https://github.com/microsoft/PowerToys/pull/2129/commits/c83a49105739bef00b69809bc161fbded80f156b